src/dragonball/src/dbs_virtio_devices/src/device.rs

```diff
@@ -220,6 +220,14 @@ where
         self.device_change_notifier.notify().map_err(Error::IOError)
     }
 
+    /// Get irq eventfd array for vritio vrings.
+    pub fn get_vring_notifier(&self) -> Vec<&EventFd> {
+        self.queues
+            .iter()
+            .map(|x| x.notifier.notifier().unwrap())
```

Comment:
Could you please help to add comments about why it's safe to call `unwrap()` here?

Reply:
Or should we return Result<Vec<&EventFd>> instead?

Reply:
This function is duplicated with get_queue_interrupt_eventfds, so I removed it.